### ISSUE TITLE
Quality of life

### ISSUE THREAD

**Issue by cytadela8** [aeternity/elixir-node#247]
Doing simple actions like sending tokens to other address or checking account balance requires writing excessive amount of code currently (multiple long function calls). Following actions should by possible with by executing short function calls:
- [x] spend tx (in #203)
- [ ] check balance
- [x] interfacing most of our modules (eg. `Chain.top_height` instead of `Aecore.Chain.Worker.top_height`)
- [ ] _any other common actions not included in this list and requiring currently multiple or complex function executions_

**Comment by cytadela8** [comment]
A week has passed without me finding time for this task. So I'm freeing it if someone wants to work on it.  I pushed one file with small progress.

**Comment by thepiwo** [comment]
creating a spend tx is included with #203 now

**Comment by thepiwo** [comment]
once #203 is merged to master I will provide an easy get balance function

**Comment by thepiwo** [comment]
maybe you should PR this for now?

**Comment by thepiwo** [comment]
there is a balance function in master already, so I guess we can close this. In case any other quality of life ideas arise, we can propose a new task
